package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "bootstrap-wysiwyg",
+  "version": "1.0.3-beta",
```

Comment:
Looks like this might throw an error now that we've released a new version. I' can probably update that and re-release.